environments/azure/flyte-core/README.md

```diff
@@ -118,6 +118,100 @@ You have a fully working Flyte environment on Azure.
 
 From this point on, you can continue your learning journey by going through the [Flyte Fundamentals](https://docs.flyte.org/en/latest/flyte_fundamentals/index.html) tutorials.
 
+## Consuming GPU accelerators
+
+To be able to request GPUs on Azure directly from your Flyte tasks, you have two main options. This section covers both and how to use them with the Terraform/OpenTofu modules:
+
+### 1. Request a generic GPU device
+
+- Go to `aks.tf` and switch the `locals.gpu_node_pool_count` value to the number of GPU-enabled nodes you need in your AKS node pool.  
+- Run a `terraform plan` + `terraform apply` operation.  
+- Save the following test workflow:
+```python
+from flytekit import ImageSpec, Resources, task
+
+image = ImageSpec(
+    base_image= "ghcr.io/flyteorg/flytekit:py3.10-1.10.2",
+     name="pytorch",
+     python_version="3.10",
+     packages=["torch"],
+     builder="envd",
+     registry="<YOUR_CONTAINER_REGISTRY>",
+ )
+# This is the image from the above image spec
+image = "<YOUR_IMAGE_REPO:TAG>"
```

Comment:
Why is this needed? Imagespec is automatically assigning `"<YOUR_IMAGE_REPO:TAG>"` when being passed to `container_image`,  no?